src/main/java/org/terasology/tutorialWorldGenerationZones/ZonedPerlinWorldGenerator.java

```diff
@@ -95,7 +94,7 @@ protected WorldBuilder createWorld() {
 
                         //A zone for the ocean, existing in between the ground height and sea level
                         .addZone(new Zone("Ocean", (x, y, z, region) ->
-                                TeraMath.floorToInt(region.getFacet(ElevationFacet.class).getWorld(x, z)) < y && y <= seaLevel)
+                                (int) Math.floor(region.getFacet(ElevationFacet.class).getWorld(x, z)) < y && y <= seaLevel)
```

Comment:
Please stick with `TeraMath.floorToInt` for the time being, this should still be available for use - eventually it might be moved to joml, though.